astrokat/simulate.py

```diff
@@ -287,27 +288,43 @@ def scan(
     def slew_time(self, target):
         """Get slew time.
 
-        How long in seconds, it took to the antennas to move
-        from previous target to the next
+        How long in seconds, it took for the antennas to move
+        from current target to the next.
+
+        Notes:
+            The slew time is calculated with consideration of az-el motion
+            of antennas instead of the current angular distance between sources,
+            assuming a slew_speed 2 deg/s. Antennas slew at 2 deg/s in az while
+            moving at 1 deg/s in el. There is about 3 seconds for overhead
 
         Parameters
         ----------
          target: katpoint.Target
              a description which contains parameters such as the
              target name, position, flux model.
 
+        Returns
+        -------
+        slew_time: float
+            The number of seconds it takes to slew
+
         """
-        slew_speed = 2.0  # degrees / sec
-        self.katpt_current.body.compute(self.katpt_current.antenna.observer)
-        target.body.compute(target.antenna.observer)
-        try:
-            separation_angle = ephem.separation(self.katpt_current.body, target.body)
-        # TODO: need to find a clean implementation with
-        # ephem_extra.StationaryBody
-        except TypeError:
```

Comment:
Is this exception no longer possible?  Do the `azel` calculations not use `body.compute` internally?

Reply:
They do not use `body.compute`